src/main/resources/narayana-checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,87 @@
+    <!--
+    <property name="cacheFile" value="${checkstyle.cache.file}"/>
+    -->
+
+    <module name="FileTabCharacter">
+        <property name="eachLine" value="true"/>
+    </module>
+
+    <module name="RegexpSingleline">
+        <!-- \s matches whitespace character, $ matches end of line. -->
+        <property name="format" value="\s+$"/>
+        <property name="message" value="Line has trailing spaces."/>
+    </module>
+
+    <module name="TreeWalker">
+
+        <!-- Checks for imports                              -->
+        <module name="AvoidStarImport">
+            <property name="allowStaticMemberImports" value="true"/>
+        </module>
+        <module name="RedundantImport"/>
+
+        <module name="UnusedImports">
+            <property name="processJavadoc" value="true"/>
+        </module>
+
+        <module name="IllegalImport">
+            <property name="illegalPkgs" value="junit.framework"/>
+        </module>
+
+        <!-- Modifier Checks                                    -->
+        <module name="ModifierOrder"/>
+        <!-- Too many errors reported since Checkstyle upgrade to > 6.2 -->
+        <module name="RedundantModifier">
+          <property name="tokens" value="ANNOTATION_FIELD_DEF"/>
+      </module>
+
+        <!-- Checks for blocks. You know, those {}'s         -->
+        <module name="LeftCurly">
+            <property name="option" value="eol"/>
+            <property name="tokens" value="CLASS_DEF,CTOR_DEF,INTERFACE_DEF,METHOD_DEF,LITERAL_SWITCH,STATIC_INIT"/>
+        </module>
+
+        <!-- Checks for common coding problems               -->
+        <!-- Disabled until http://sourceforge.net/tracker/?func=detail&aid=2843447&group_id=29721&atid=397078 is fixed-->
+        <!--<module name="DoubleCheckedLocking"/>-->
+        <module name="EmptyStatement"/>
+        <module name="IllegalInstantiation"/>
+        <!-- removed in checkstyle 6.2, see https://github.com/checkstyle/checkstyle/issues/473 -->
+        <!--module name="RedundantThrows">
+            <property name="allowUnchecked" value="true"/>
+            <property name="suppressLoadErrors" value="true"/>
+        </module-->
+
+        <!-- Miscellaneous other checks.                   -->
+        <module name="UpperEll"/>
+        <module name="PackageAnnotation"/>
+        <!-- This check doesn't actually work correctly if the overloaded equals is an abstract method on a base class -->
+        <!-- <module name="CovariantEquals"/> -->
+        <module name="ArrayTypeStyle"/>
+
+        <!-- This is required for SuppressWarningsFilter to work. -->
+        <module name="SuppressWarningsHolder"/>
+
+        <!-- Disallow @SuppressWarnings("all") -->
+        <module name="SuppressWarnings">
+            <property name="format" value="^all$"/>
+        </module>
+    </module>
+
+    <module name="SuppressionFilter">
+        <property name="file" value="/wildfly-checkstyle/suppressions.xml" />
```

Comment:
this was wrong in my PR probably, the path should be `/narayana-checkstyle/suppressions.xml`